internal/fs/wrappers/monitoring.go

```diff
@@ -42,7 +54,20 @@ func init() {
 			Aggregation: view.Sum(),
 			TagKeys:     []tag.Key{tags.FSOp, tags.FSError},
```

Comment:
Are we logging error types also? If yes can we get error count per operation from that?